openquake/engine/config.py

```diff
@@ -143,10 +142,8 @@ def abort_if_no_config_available():
         msg = (
             "\nYou are not authorized to read any of the OpenQuake "
             "configuration files.\n"
-            "Please contact a system administrator or run the following "
-            "command:\n\n"
-            "   sudo gpasswd --add %s openquake"
-            % pwd.getpwuid(os.geteuid()).pw_name)
+            "Please check permissions on the configration files in %s."
+            % cfg._get_paths())
```

Comment:
While you are at it, you could fix the mispelling `configration` -> `configuration`